jumpscale/sals/reservation_chatflow/solutions.py

```diff
@@ -307,6 +307,16 @@ def cancel_solution(self, solution_wids):
         for wid in ids_to_delete:
             j.sals.zos.workloads.decomission(wid)
 
+    def cancel_solution_uuid(self, solution_uuid):
+        ids_to_delete = []
+        # Get workloads with specific UUID
+        for workload in j.sals.zos.workloads.list(j.core.identity.me.tid, next_action="DEPLOY"):
+            if solution_uuid == self.get_solution_uuid(workload):
+                ids_to_delete.append(workload.id)
```

Comment:
Why do we append wids and then loop to delete? Shouldn't we delete wid directly on this line?